packages/strapi-plugin-content-manager/admin/src/components/SelectMany/ListItem.js

```diff
@@ -0,0 +1,77 @@
+import React, { memo, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { useDrag, useDrop } from 'react-dnd';
+import { getEmptyImage } from 'react-dnd-html5-backend';
+
+import pluginId from '../../pluginId';
+import ItemTypes from '../../utils/ItemTypes';
+
+import { Li } from './components';
+import Relation from './Relation';
+
+function ListItem({
+  data,
+  findRelation,
+  mainField,
+  moveRelation,
+  nextSearch,
+  onRemove,
+  targetModel,
+}) {
+  const to = `/plugins/${pluginId}/${targetModel}/${data.id}?redirectUrl=${nextSearch}`;
+
+  const originalIndex = findRelation(data.id).index;
+  const [{ isDragging }, drag, preview] = useDrag({
+    item: {
+      type: ItemTypes.RELATION,
+      id: data.id,
+      originalIndex,
+      data,
+      mainField,
+    },
+    collect: monitor => ({
+      isDragging: monitor.isDragging(),
+    }),
+  });
+  const [, drop] = useDrop({
+    accept: ItemTypes.RELATION,
+    canDrop: () => false,
+    hover({ id: draggedId }) {
+      if (draggedId !== data.id) {
+        const { index: overIndex } = findRelation(data.id);
+        moveRelation(draggedId, overIndex);
+      }
+    },
+  });
+
+  useEffect(() => {
+    preview(getEmptyImage(), { captureDraggingState: true });
+  }, [preview]);
+
+  const opacity = isDragging ? 0.2 : 1;
```

Comment:
Well since the displayed dragged component for the select is different than the group which is also different than the one for the layout I need to add the same condition to each single one of them. I don't see how I can manage that with a css class or a styled component.